README.md

```diff
@@ -143,14 +143,20 @@ The client's configured. Now return to the Director to add the client profile,
 cd /etc/bacula/local.d/servers/
 cp 1/self.conf 2/server-1.conf
 nano 2/server-1.conf
-# Edit Name = self, Password = "XYZ", Address = "127.0.0.1"
-# New configuration should look like
-# Client {
-#        Name = server-1
-#        Password = "foo/bar+baz"
-#        Address = "43.2.1.5"
-#        FileSet = "Client-Layer"
-# }
+```
+
+Update Name, Password, and Address with information of the client you wish to backup. Example:
+
+```bash
```

Comment:
Remove "bash" syntax highlighting type. This was commented out to disambiguate from commands, but setting aside is adequate.